bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -122,6 +123,22 @@ trait UtilBijections {
       override def apply(pool: FuturePool) = new TwitterExecutionContext(pool)
       override def invert(context: ExecutionContext) = new ScalaFuturePool(context)
     }
+
+  object Owned {
```

Comment:
we now have objects Owned and shared under twitter_util, the namespacing there doesn't make too much sense. A sub package or object to put them into a Buf namespace so it aligns more with util I think would be good here